course3/homework-list/list.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include <cstddef>
+
+
+namespace Karun
+{
+
+template<typename T>
+class List
+{
+public:
+	class Node
+	{
+	friend class List;
+	
+	public:
+		Node() = default;
+		Node(const Node& node) = default;
```

Comment:
Fixed. This is a remnant of me initially returning Node objects instead of iterators.